pom.xml

```diff
@@ -0,0 +1,448 @@
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+  -->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <name>Fail Pipeline Sink Plugins</name>
+  <groupId>co.cask.hydrator</groupId>
+  <artifactId>failpipeline-sink-plugins</artifactId>
+  <modelVersion>4.0.0</modelVersion>
+  <version>1.6.0-SNAPSHOT</version>
+
+  <licenses>
+    <license>
+      <name>The Apache Software License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+      <comments>A business-friendly OSS license</comments>
+    </license>
+  </licenses>
+
+  <developers>
+    <developer>
+      <name>Cask Data</name>
+      <email>cask-dev@googlegroups.com</email>
+      <organization>Cask Data, Inc.</organization>
+      <organizationUrl>http://www.cdap.io</organizationUrl>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:https://github.com/hydrator/failpipeline-sink.git</connection>
+    <developerConnection>scm:git:git@github.com:hydrator/failpipeline-sink.git</developerConnection>
+    <url>https://github.com/hydrator/failpipeline-sink.git</url>
+    <tag>HEAD</tag>
+  </scm>
+
+  <distributionManagement>
+    <repository>
+      <id>sonatype.release</id>
+      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
+    </repository>
+    <snapshotRepository>
+      <id>sonatype.snapshots</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+    <site>
+      <id>cask</id>
+      <url>http://cask.co</url>
+    </site>
+  </distributionManagement>
+
+  <issueManagement>
+    <url>https://issues.cask.co/browse/HYDRATOR</url>
+  </issueManagement>
+  <properties>
+    <main.basedir>${project.basedir}</main.basedir>
+    <cdap.version>4.1.0-SNAPSHOT</cdap.version>
+    <hadoop.version>2.3.0</hadoop.version>
+    <junit.version>4.11</junit.version>
+    <hydrator.version>1.4.0</hydrator.version>
+    <widgets.dir>widgets</widgets.dir>
+    <docs.dir>docs</docs.dir>
+    <app.parents>system:cdap-etl-batch[4.1.0-SNAPSHOT,4.2.0-SNAPSHOT),system:cdap-etl-realtime[4.1.0-SNAPSHOT,4.2.0-SNAPSHOT),system:cdap-data-pipeline[4.1.0-SNAPSHOT,4.2.0-SNAPSHOT),system:cdap-data-streams[4.1.0-SNAPSHOT,4.2.0-SNAPSHOT)</app.parents>
+    <!-- this is here because project.basedir evaluates to null in the script build step -->
+    <main.basedir>${project.basedir}</main.basedir>
+  </properties>
+
+  <repositories>
+    <repository>
+      <id>project.local</id>
+      <name>project</name>
+      <url>file:${project.basedir}/repo</url>
+    </repository>
+  </repositories>
+
+  <dependencies>
```

Comment:
Seems like an large number of dependencies for such a simple plugin. are you sure all of them are needed?